userguide/tutorials/debugging.adoc

```diff
@@ -532,8 +532,18 @@ In your message, specify:
 kpm diagnostic --killbill-api-credentials=bob lazar \
                --killbill-credentials=admin password \
                --killbill-url=http://127.0.0.1:8080 \
-               --killbill-web-path=/var/lib/tomcat7/webapps/ROOT/ \
-               --kaui-web-path=/var/lib/tomcat7/webapps/ROOT/ \
-               --log-dir=/var/lib/tomcat7/logs \
-               --account_export=ACCOUNT_ID
+               --killbill-web-path=/var/lib/tomcat/webapps/ROOT.war \
+               --log-dir=/var/lib/tomcat/logs \
+               --account-export=ACCOUNT_ID
+----
+
+If you are using our standard Docker images, you can simply run:
+
+[source,bash]
+----
+$KPM_DIAGNOSTIC_CMD -e killbill_user=admin \
```

Comment:
Confused what is this env variable `KPM_DIAGNOSTIC_CMD` -- is it defined above in the doc ? Also is this supposed to be run from inside or outside the container ?

Reply:
I've clarified it in 47dd2748 - the variable is defined in our Dockerfile and abstracts the right default properties.